sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.aest.controller.manualReassessment.e2e-spec.ts

```diff
@@ -0,0 +1,233 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentAppeal,
+  createFakeStudentAppealRequest,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import {
+  ApplicationStatus,
+  AssessmentTriggerType,
+  NoteType,
+  StudentAssessmentStatus,
+} from "@sims/sims-db";
+
+describe("AssessmentAESTController(e2e)-manualReassessment", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should create a manual reassessment when requested.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource);
+    application.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.Completed;
+    const originalAssessment = application.currentAssessment;
+    application.studentAssessments = [application.currentAssessment];
```

Comment:
This line is not supposed to be necessary. Is there a need to keep it?
```tts
application.studentAssessments = [application.currentAssessment];
```

Reply:
Yes, it is needed because the original assessment is got from `application.studentAssessments`.

Reply:
Yes, after we talked I see that you're right and removed it. Thanks.